.github/workflows/validate_pyinstaller.yml

```diff
@@ -0,0 +1,57 @@
+name: Validate Pyinstaller Build
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build-for-linux:
+    name: build-pyinstaller-linux
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64:latest
+    if: github.repository_owner == 'aws'
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build PyInstaller
+        run: |
+          chmod +x ./installer/pyinstaller/build-linux.sh
+          CI_OVERRIDE=1 ./installer/pyinstaller/build-linux.sh aws-sam-cli-linux-x86_64.zip
+      - name: Basic tests for PyInstaller
+        run: |
+          unzip .build/output/aws-sam-cli-linux-x86_64.zip -d sam-installation
+          ./sam-installation/install
+          sam-beta --version
```

Comment:
Not blocking: can we kick off another suite of tests that is bare bones across the commands, such that each command is atleast invoked once? could be a follow up commit. Could be a first step towards artifact based testing.

Reply:
I do agree with this. But I would rather have that as a separate PR and include the tests that we want to run with the pyinstaller. So that we can incrementally improve our installer test coverage.